src/lib/wallet/GoodWalletClass.js

```diff
@@ -368,24 +372,30 @@ export class GoodWallet {
 
   async getAmountAndQuantityClaimedToday(): Promise<any> {
     try {
-      const res = (await this.UBIContract.methods.getDailyStats().call()) || [ZERO, ZERO]
-      return {
-        people: res[0].toNumber(),
-        amount: res[1].toNumber(),
+      let stats = await this.UBIContract.methods.getDailyStats().call()
+
+      if (!stats) {
+        stats = [ZERO, ZERO]
       }
+
+      const { amount, count } = mapValues(stats, (value, key) =>
```

Comment:
complicated....
return {people: res ? res[0].toNumber() : 0 ... }

Reply:
res[0] looks weird, destructuring is much prettier

how about 
```
import { invokeMap } from 'lodash'

const [people, amount] = invokeMap(stats, 'toNumber')

return { amount, count }
```
?

looks pretty & simple

Reply:
invokeMap will convert {0: <BN>, 1: <BN>} object to array, i've checked it

Reply:
@sirpy  pls look at https://github.com/GoodDollar/GoodDAPP/pull/1596/commits/f9c4efe35a30e5edcee7cc24cdf0eeb31426c854#diff-70a52ee4a3dcebf732957d95df85685aR375